src/main/java/genji/Parser.java

```diff
@@ -0,0 +1,133 @@
+package genji;
+
+import genji.command.Command;
+import genji.command.AddCommand;
+import genji.command.DateCommand;
+import genji.command.DeleteCommand;
+import genji.command.ExitCommand;
+import genji.command.FindCommand;
+import genji.command.ListCommand;
+import genji.command.MarkCommand;
+import genji.command.UnmarkCommand;
+import genji.task.ToDo;
+import genji.task.Deadline;
+import genji.task.Event;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
+
+/**
+ * A class to parse user's commands
+ */
+public class Parser {
+
+    /**
+     *Parses user's command into different types of commands
+     * Creates the task if the command ask to
+     * @param command user's input
+     * @return The corresponding command according to user's input
+     * @throws GenjiException When user's command is invalid
+     */
+    public static Command parse(String command) throws GenjiException {
+        if (command.equals("bye")) {
+            return new ExitCommand();
+        } else if (command.equals("list")) {
```

Comment:
Good use of Egyptian brackets, proper indentations and spacings! The code style has been well-followed 👍